ql/src/test/org/apache/hadoop/hive/metastore/txn/TestTxnHandler.java

```diff
@@ -244,8 +245,16 @@ public void testAbortTxns() throws Exception {
     List<Long> txnList = openedTxns.getTxn_ids();
     txnHandler.abortTxns(new AbortTxnsRequest(txnList));
 
+    OpenTxnRequest replRqst = new OpenTxnRequest(2, "me", "localhost");
+    replRqst.setReplPolicy("default.*");
+    replRqst.setReplSrcTxnIds(Arrays.asList(1L, 2L));
+    List<Long> targetTxns = txnHandler.openTxns(replRqst).getTxn_ids();
+    txnHandler.abortTxns(new AbortTxnsRequest(targetTxns));
+
+    assertFalse(targetTxnsPresentInReplTxnMap(targetTxns));
```

Comment:
Test REPL_TXN_TIMEOUT case as well

Reply:
This test is included in TestDbTxnManager.

Reply:
This test is included in TestDbTxnManager.